src/Resources/views/Order/Grid/Field/number.html.twig

```diff
@@ -0,0 +1 @@
+<a href="{{ path('sylius_admin_order_show', {'id' : sylius_refund_get_order_id_by_order_number(data.orderNumber) })|raw }}">#{{ data.orderNumber }}</a>
```

Comment:
I'm not really convinced to the idea of having a template function `sylius_refund_get_order_id_by_order_number` access the repository every time it has to show order number link just to generate the link itself.

We have an endpoint that shows order page for given order ID, what if we could create an endpoint that takes an order number, gets the order ID and redirects you to the former? This way we do not need any additional SQL queries for every order link (20-100 per page). What do you think? This has to be done in Sylius/Sylius though, but I think we're in process of making shortcuts like this there also.

Reply:
Actually, we can temporarily create that action in this plugin, if there's no need for it in Sylius itself.

Reply:
You are a right ! but it must be in Sylius/Sylius. In this moment I think that I haven't better way

Reply:
I changed the twig function on redirect because twig function when render link in every row send query to database, now db is used when clicking in link. Thanks for advice